src/common/databaseFilters.ts

```diff
@@ -8,15 +8,16 @@ export const fullNameSearch = <DbData extends DbClientRow | DbParcelRow>(
     clientIsActiveColumnLabel: Extract<keyof DbData, "is_active" | "client_is_active">
 ): ServerSideFilterMethod<DbData, string> => {
     return (query, state) => {
-        if (state === "") {
+        const newState = state.replace(/[,.():/]/g, "");
```

Comment:
Can we call this something like `cleanState` to better reflect its purpose?

Reply:
changed into alphanumeric input only and the filter I made changes to were the base filters, so the client page applied as well